src/plugins/commit-message/index.js

```diff
@@ -30,13 +65,30 @@ const EXCLUDED_REPOSITORY_NAMES = new Set([
  */
 function checkCommitMessage(message) {
     const commitTitle = message.split(/\r?\n/)[0];
+    const errCode = [];
 
     if (message.startsWith("Revert \"")) {
-        return true;
+        return [true, []];
     }
 
+    let isValid = true;
+
     // First, check tag and summary length
-    let isValid = TAG_REGEX.test(commitTitle) && commitTitle.length <= MESSAGE_LENGTH_LIMIT;
+    if (!TAG_REGEX.test(commitTitle)) {
```

Comment:
Can we be a bit smarter about this? If someone uses "new:", they might not notice that the first letter wasn't uppercase. I'd rather have an explicit messages saying the tag must begin with an uppercase letter.